### PR TITLE
Users/pritamd/update zoom2

### PR DESCRIPTION
**Pull request by pritamDas9191**
On content load, the default behavior is to persist the zoom settings from previous media loaded.
This can be reset by using following Api on ZoomManager.

*CT Library Changes*
* `resetZoomOnContentLoad` is introduced on `ZoomManager` so that the zoom behavior can be reset before new content load.

*Unit Tests*
* Jest and ts-jest is setup now in this repo.
* Added Unit tests for `ZoomManager` file.

### CODE REVIEW

File: src/canvastools/ts/CanvasTools/CanvasTools.Editor.ts [hunk 1]
```diff
@@ -887,50 +893,72 @@ export class Editor {
...
-                } else {
-                    this.editorDiv.style.width = `${scaledFrameWidth}px`;
-                }
+    /**
+     * Helper function to zoom the editor to given scale.
+     */
+    private zoomEditorToScale(scaledFrameWidth: number, scaledFrameHeight: number): void {
+        if (!this.editorContainerDiv.offsetWidth) {
+            this.editorContainerDiv = <HTMLDivElement>document.getElementsByClassName("CanvasToolsContainer")[0];
+            this.editorDiv = <HTMLDivElement>document.getElementsByClassName("CanvasToolsEditor")[0];
```
Comment (soyfestivo) [inline src/canvastools/ts/CanvasTools/CanvasTools.Editor.ts]: Does this work? This is syntax I have not see in ts before. Wouldn't be be better to say:
```typescript
this.editorDiv = document.getElementsByClassName("CanvasToolsEditor")[0] as HTMLDivElement;
```

File: src/canvastools/ts/CanvasTools/Core/ZoomManager.spec.ts [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+        let zoomManagerInstance = getZoomManager();
+        expect(zoomManagerInstance).toBeDefined();
+        expect(zoomManagerInstance.isZoomEnabled).toBeTruthy();
+        let zoomData = zoomManagerInstance.getZoomData();
+        expect(zoomData.minZoomScale).toEqual(1);
+        expect(zoomData.maxZoomScale).toEqual(maxZoom);
+        zoomManagerInstance.deleteInstance();
+    });
+
+    it("check if the zoom in and zoom out works properly", () => {
```
Comment (soyfestivo) [inline src/canvastools/ts/CanvasTools/Core/ZoomManager.spec.ts]: :thumbsup:

Comment (soyfestivo) [reply src/canvastools/ts/CanvasTools/Core/ZoomManager.spec.ts]: Thanks for writing tests!

**Comment by pritamDas9191** [comment]
The sign off was already provided.

**Pull request merged**